### ISSUE TITLE
xLights 2024.14 crashing & freezing 

### ISSUE THREAD

**Issue by Melinda738** [xLightsSequencer/xLights#4730]
I have not been able to replicate the behavior with one consistent action, though moving the effects grid screens slightly up or down seems to trigger freezing even without a sequence open (and often crash). It also freezes for up to 3 minutes when auto save is on - tried turning that off, but still crashing. It wasn't crashing as frequently when using my normal home show folder, but using a temporary show folder layout has been crashing every ~5 minutes. Before each crash there is a huge amount of [WARN] Model::InitRenderBufferNodes BufferWi was overly large ... overridden to be 10000. Re-naming a timing track & saving did seem to trigger a crash but I don't think that was the same as all the other crashes - might be unrelated

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)
1. Go to '...' It can happen when modifying an effect or just playing a sequence, nothing consistent - the freezing will happen when adjusting the effects grid windows to be smaller or larger. 

**Expected behavior**
normal xlights function of whatever i'm doing

**Screenshots**

2024-08-18 10:15:52,917 19420 log_base [WARN] Model::InitRenderBufferNodes BufferWi was overly large ... overridden to be 10000.
2024-08-18 10:15:52,917 19420 log_base [WARN] Model::InitRenderBufferNodes BufferWi was overly large ... overridden to be 10000.
2024-08-18 10:15:52,936 19420 log_base [DEBUG] Job pool new size 5.
2024-08-18 10:16:16,606 3628 log_base [INFO] Start Time: 2024-08-18_10-16-16-909.
2024-08-18 10:16:16,606 3628 log_base [INFO] Log4CPP config read from xlights.windows.properties.
2024-08-18 10:16:16,606 3628 log_base [INFO] Current working directory C:\Program Files\xLights.
2024-08-18 10:16:16,606 3628 log_base [INFO]      : ALERT CRIT DEBUG EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_base : ALERT CRIT DEBUG EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_conversion : ALERT CRIT DEBUG EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_curl : ALERT CRIT EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_generatecustommodel : ALERT CRIT DEBUG EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_jobpool : ALERT CRIT EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_opengl : ALERT CRIT EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_opengl_trace : ALERT CRIT EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_pianodata : ALERT CRIT DEBUG EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_prepcustommodel : ALERT CRIT DEBUG EMERG ERROR FATAL INFO NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_render : ALERT CRIT EMERG ERROR FATAL NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_rendercache : ALERT CRIT EMERG ERROR FATAL WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_timer : ALERT CRIT EMERG ERROR FATAL WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_work : ALERT CRIT EMERG ERROR FATAL WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO]     log_zcpp : ALERT CRIT EMERG ERROR FATAL NOTICE WARN 
2024-08-18 10:16:16,606 3628 log_base [INFO] ******* OnInit: XLights started.
2024-08-18 10:16:16,606 3628 log_base [INFO] Version: 2024.14
2024-08-18 10:16:16,606 3628 log_base [INFO] Bits: 64bit
2024-08-18 10:16:16,606 3628 log_base [INFO] Build Date: Aug  6 2024
2024-08-18 10:16:16,606 3628 log_base [INFO] WX Version: wxWidgets 3.3.0
2024-08-18 10:16:16,606 3628 log_base [INFO] Machine configuration:
2024-08-18 10:16:16,607 3628 log_base [INFO]   Total memory: 65536 MB
2024-08-18 10:16:16,607 3628 log_base [INFO]   Free Memory: 60632702976.
2024-08-18 10:16:16,607 3628 log_base [INFO]   Current directory: C:\Program Files\xLights

**Versions (please complete the following information):**
 - OS: Windows 11
 - xLights version 2024.14

**Additional context**
the last version that worked for me was 2022.14 every other version (I tried -which wasn't many) has crashed since. When updating to version 2024.10 (or 11 I cant remember) I used clean up file folders and then because of the crashing I tried to revert back to 2022.14, but it would no longer recognise one of my show folders names "Xlights Exports" it would only load the first word xlights and then of course could not open the folder -nothing fancy in the name but now I have no option but to use current versions. Other show folders would open fine but not the one I need for all my sequences.
Historically my render times have been terrible and lots of memory being used despite 64gb memory available- the buffer change seems to have helped but is now crashing instead. 

**Attachments**
Happy to share a Drive link to crash files with Devs but not publicly as they contain paid sequences.

If you would prefer not to post your sequences publicly please say so and be ready to PM them to one of the developers when asked.

**Comment by derwin12** [comment]
Are you able to grab the nightly to see if that helps? The overly large buffer was addressed in a commit.

**Comment by derwin12** [comment]
Crashed since "2022.14" - wow .. so anything in the past two years could be the cause?

**Comment by Melinda738** [comment]
> Are you able to grab the nightly to see if that helps? The overly large buffer was addressed in a commit.

Thanks Darryl that's great news! - where do I go to get the nightly?  I don't remember.

**Comment by cybercop23** [comment]
This...
https://github.com/xLightsSequencer/xLights/actions/runs/10437975387/artifacts/1824065858

**Comment by Melinda738** [comment]
Hi, it seemed to last a bit longer this time - I got a solid 30 mins! Possibly a driver issue?

![Screenshot 2024-08-19 180451](https://github.com/user-attachments/assets/67c6dcbb-9905-4862-9ebb-da5a56a805d0)
![Screenshot 2024-08-19 180138](https://github.com/user-attachments/assets/11d75184-612e-4167-8f03-4c370c52f276)
![Screenshot 2024-08-19 175817](https://github.com/user-attachments/assets/96d5ffb7-7c3b-4f87-9
[19082024_xLights_l4cpp.log](https://github.com/user-attachments/files/16657622/19082024_xLights_l4cpp.log)
1ff-a4be0026cd54)

**Comment by derwin12** [comment]
Perhaps Dan or Keith are getting crash dumps - I dont see one being generated in the above log though. Is this something repeatable with a certain sequence or any other type of triggers? if so send us a package sequence.

**Comment by blun23** [comment]
So everyone knows that I know little about computers.  But I can trouble shoot with the best of em.  What type of Pc are you running Melinda?  How large is your show?

**Comment by derwin12** [comment]
google-eeze seems to point to the video driver. Under preferences turn off Other->Hardware Video decoding .. or perhaps try a different setting there. Of course be sure you have the latest nvidia drivers in place as well.
https://www.vegascreativesoftware.info/us/forum/fault-mode-crash-igxelpicd64-dll--137659/

**Comment by Melinda738** [comment]
Thanks for the replies! I've opened a support case with Gigabyte. The NVIDIA driver is July 2024 but it looks like Gigabyte themselves may want me to use a 2021 driver. I've just tried going to an older version of the Intel driver (recommended on the Gigabyte page) so will see if that makes any difference. The crashing only happens with xLights all other applications work fine. I had already tried turning video decoding off as all of my sequences use videos - I have tried disabling render on the videos - no joy there either. Unfortunately I need the videos playing because I match the colours for the sequence effects from the videos. The buffer messages have gone from the logs now :-) I'll do some more testing and come back with what I find - rather than waste everyone's time if it's just a driver issue. 

![Screenshot 3](https://github.com/user-attachments/assets/213167d4-01ec-42d9-91a9-0caf0305e2ae)

![Screenshot 2](https://github.com/user-attachments/assets/70747cfe-dded-4114-8ddb-58d793f25c61)
